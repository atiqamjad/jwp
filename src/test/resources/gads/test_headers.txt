<br>
<BR />
	<p>One - line paragraph.</p>
   <P>One two?</p>


<p>One, two
three four</p>
<p>One two, three four five!</p>
<p>One two, three four 5 six seven eight nine not wrap h3 tag!</p>
<br  />
<br/ >
<br>
OEF