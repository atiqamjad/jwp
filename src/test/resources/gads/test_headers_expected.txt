<h3>One - line paragraph.</h3>

<h3>One two?</h3>

<h3>One, two three four</h3>

<h3>One two, three four five!</h3>

One two, three four 5 six seven eight nine not wrap h3 tag!

OEF